types/react-scrollable-anchor/index.d.ts

```diff
@@ -0,0 +1,26 @@
+// Type definitions for react-scrollable-anchor 0.6
+// Project: https://github.com/gabergg/react-scrollable-anchor
+// Definitions by: Antoine DOUBOVETZKY <https://github.com/AntoineDoubovetzky>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.8
+
+import * as React from 'react';
+
+export interface ScrollableAnchorProps {
+  id: string;
+  children?: React.ReactNode;
+}
+
+declare const ScrollableAnchor: React.ComponentType<ScrollableAnchorProps>;
+
+export interface ConfigureAnchorsOptions {
+  offset?: number;
+  scrollDuration?: number;
+  keepLastAnchorHash?: boolean;
+}
+
+export default ScrollableAnchor;
```

Comment:
Can you please instead write this as `export default class ScrollableAnchor extends React.ComponentType<ScrollableAnchorProps>`

Reply:
@sheetalkamat 
It is updated !
What difference does it make exactly ? I'm asking to make sure not to make the mistake again :)